docs/VictoriaLogs/README.md

```diff
@@ -15,6 +15,7 @@ It provides the following key features:
 - VictoriaLogs capacity and performance scales lineraly with the available resources (CPU, RAM, disk IO, disk space).
   It runs smoothly on both Raspberry PI and a server with hundreds of CPU cores and terabytes of RAM.
 - VictoriaLogs can handle much bigger data volumes than ElasticSearch and Grafana Loki when running on comparable hardware.
+  See benchmark with ElasticSearch [here](https://github.com/VictoriaMetrics/VictoriaMetrics/tree/master/deployment/logs-benchmark).
```

Comment:
we should define a separate section with the title Benchmark (we definitely want to provide an anchored link in the future) and add a description

We encourage you to run benchmarks on your own. Please share the results with us just dropping the line any of our Community channels[link]
However, we prepared benchmark suite that covers ElasticSearch and VictoriaLogs